plugin/build.gradle

```diff
@@ -293,7 +293,7 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.profile.MLPredictRequestStats',
         'org.opensearch.ml.action.deploy.TransportDeployModelAction',
         'org.opensearch.ml.action.deploy.TransportDeployModelOnNodeAction',
-	'org.opensearch.ml.action.undeploy.TransportUndeployModelsAction',
+	    'org.opensearch.ml.action.undeploy.TransportUndeployModelsAction',
```

Comment:
Seems wrong indent

Reply:
Not quite sure why it's showing like that, but in my end, it's maintaining the correct indent.